conf/messages.en

```diff
@@ -211,5 +212,6 @@ charityUKAddress.officialAddress.outsideUK.link = Charity’s official address i
 # CharityContactAddressChoicePage Messages
 # ----------------------------------------------------------
 isCharityOfficialAddressInUK.title = Is your charity''s official address in the UK?
+isCharityOfficialAddressInUK.section = Charity Information
```

Comment:
same as above... keep one message key for section

Reply:
Done as per comment